.github/workflows/new-release.yml

```
@@ -4,40 +4,23 @@ on:
     tags:
       - v[0-9]+.[0-9]+.[0-9]+
 
+env:
+  GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+  GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+  GITHUB_KEY: ${{ secrets.GithubKey }}
```

This isn't quite right, I thought that I changed this. It should be:
```suggestion
  GITHUB_KEY: ${{ secrets.GITHUB_TOKEN }}
```

---

The 2 keys are both `GITHUB_TOKEN`?

https://github.com/JLLeitschuh/ktlint-gradle/blob/ddd465e28d77b879384886e1eef5666ebe518b4d/.github/workflows/new-release.yml#L36-L42

---

If not, I need to place these 2 `GITHUB_KEY` under action.